src/app/search/items/search-filters/known-values.ts

```diff
@@ -228,6 +229,25 @@ const knownValuesFilters: ItemFilterDefinition[] = [
       return (item) => breakerType.includes(item.breakerType?.hash as BreakerTypeHashes);
     },
   },
+  {
+    keywords: 'breakermod',
+    description: tl('Filter.BreakerMod'),
+    format: 'query',
+    suggestions: Object.keys(breakerTypes),
+    destinyVersion: 2,
+    filter: ({ filterValue }) => {
+      const breakerType: BreakerTypeHashes[] | undefined =
+        breakerTypes[filterValue as keyof typeof breakerTypes];
+      if (!breakerType) {
+        throw new Error(`Unknown breaker type ${breakerType as string}`);
+      }
+      const breakingIchs = breakerType.flatMap((ty) => artifactBreakerMods[ty] || []);
```

Comment:
The tricky thing here is that not all of the artifact breaker mods may be active. Do we want to care about that? I assume most folks enable all the breaker mods, at least the basic ones, but then there's always some later column one that I ignore or haven't unlocked...

The tricky part of course being that each character has their own artifact.

Reply:
All the options have problems, but I find thinking "the principle of least surprise": Which model is easiest for users to recognize and understand?

To me, that's the one where the filter is showing potentially-useful stuff, rather than "works on this character, but not on that character", or "worked yesterday, but not today", etc.